core/src/View/Kokkos_BasicView.hpp

```diff
@@ -385,23 +385,67 @@ class BasicView {
   }
 
  public:
+  // check for accessor_arg for managed view ctors
+  template <class... P>
+  explicit inline BasicView(
```

Comment:
The `inline` specifier here is pointless (I see we have it at other places so do whatever you want)

Reply:
Removed